elements/lisk-api-client/src/ws_channel.ts

```diff
@@ -131,6 +152,12 @@ export class WSChannel {
 		};
 
 		const send = new Promise((resolve, reject) => {
+			if (!isNode) {
+				this._ws?.send(JSON.stringify(request));
+				resolve();
+				return;
+			}
+
```

Comment:
in invoke, the send is not meant to be acknowledge timeout, the `Response not received in X` should be the acknowledge timeout.
ie) send it self doesn't need the promise

Reply:
Done